sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -87,10 +79,11 @@ object AlluxioUtils extends Logging with Arm {
   private var alluxioMasterPort: Option[Int] = None
```

Comment:
why do we still have this if its in the AlluxioFS?  Couldn't we change to get from there and you could mock it wihtout reset?

Reply:
Done.